docs/deploying-airbyte/on-gcp-compute-engine.md

```diff
@@ -62,10 +62,8 @@ sudo usermod -a -G docker $USER
 7. Install `docker-compose` on your VM instance by following the below commands in your VM terminal:
 
 ```bash
-sudo apt-get -y install wget
-sudo wget https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m) -O /usr/local/bin/docker-compose
-sudo chmod +x /usr/local/bin/docker-compose
-docker-compose --version
+sudo apt-get -y install docker-compose-plugin
```

Comment:
Do we need to run that on our google compute instance for cart if we want to bump their platform version.

Reply:
Yes we do